config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-presubmits-main.yaml

```diff
@@ -135,7 +135,7 @@ presubmits:
     always_run: false
     optional: true
     decorate: true
-    run_if_changed: '^((exp)/|main\.go|go\.mod|go\.sum|Dockerfile|Makefile)'
+    run_if_changed: '^((exp)/|main\.go|go\.mod|go\.sum|managed.*\.go|Dockerfile|Makefile)'
```

Comment:
could you please also update the equivalent jobs in release branches? eg. https://github.com/kubernetes/test-infra/blob/a12543a7a69cef0d4e852f2fcc6e8c342ff3947c/config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-presubmits-release-v1beta1.yaml#L176 (there should also be a v1alpha3 and v1alpha4 one)